opensciencegrid/mailchimp-sync/Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG BASE_YUM_REPO=release
+ARG BASE_OSG_SERIES=3.6
+
+FROM opensciencegrid/software-base:$BASE_OSG_SERIES-el8-$BASE_YUM_REPO
+
+LABEL maintainer OSG Software <help@opensciencegrid.org>
+
+RUN yum -y install \
+             vim && \
+    yum -y clean all
+
+ADD mailchimp-sync /srv/mailchimp-sync
+
+ENTRYPOINT ["/srv/mailchimp-sync/runme.sh"]
```

Comment:
```suggestion
CMD ["/srv/mailchimp-sync/runme.sh"]
```

Reply:
No need to use ENTRYPOINT here; you're not trying to wrap all commands, nor do you need to override an inherited entrypoint.